### PR TITLE
Added new metric AUC

### PR DESCRIPTION
**Pull request by marymartin16**

### CODE REVIEW

File: R/AUC.R [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#' two columns: subject id and corresponding hourly average AUC value is returned.
+#'
+#' @export
+#'
+#' @details
+#' A tibble object with 1 row for each subject, a column for subject id and
+#' a column for hourly average AUC values is returned. NA glucose values are
+#' omitted from the calculation of the AUC.
+#'
+#' AUC is calculated using the formula: (dt0/60) * ((gl[2:length(gl)] + gl[1:(length(gl)-1)])/2),
```
Comment (irinagain) [inline R/AUC.R:28]: Can you please add that this is trapezoidal rule?

File: R/AUC.R [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+#'
+#' @usage
+#' auc(data, tz="")
+#'
+#' @param data DataFrame object with column names "id", "time", and "gl", or numeric vector of glucose values.
+#' @param tz String value of time zone
+#'
+#' @return
+#' If a data.frame object is passed, then a tibble object with
+#' two columns: subject id and corresponding hourly average AUC value is returned.
```
Comment (irinagain) [inline R/AUC.R:14]: I have a questions on what is meant by hourly average AUC values. If my understanding is correct, AUC is calculated for every hour out of 24 hour period, correct? And then they are averaged across days? Just making sure I understand this right. Perhaps a little bit more explanation can be given. Thank you!

**Review by irinagain — COMMENTED** [review]
See comments in text

File: R/AUC.R (hunk 1 above)
Comment (marymartin16) [reply R/AUC.R:28]: I have added the trapezoidal rule.

File: R/AUC.R (hunk 2 above)
Comment (marymartin16) [reply R/AUC.R:14]: Please let me know if more explanation is needed. Thank you for your comments!

**Pull request merged**